### DISCUSSION TITLE
Live Coding: How to Develop WebRTC iOS apps? - May 19 , 2:00 – 2:30pm GMT+3

### DISCUSSION THREAD

**Discussion by mekya** [ant-media/Ant-Media-Server#5121] [Show and tell]
Are you an iOS developer looking to create real-time WebRTC iOS streaming applications? Building a WebRTC iOS application has never been easier!

In this quick event, we’ll show you how to create an iOS app project in Xcode, add the WebRTC-iOS-SDK dependency, and publish and play WebRTC live streams in just a few simple steps.

We'll walk through this blog post
[4 Simple Steps to Build WebRTC iOS Apps and Stream Like a Pro](https://www.google.com/url?q=https://antmedia.io/build-and-try-webrtc-ios-app-for-dummies/&sa=D&source=calendar&ust=1684864756002556&usg=AOvVaw06aoUYieMZo6lRqakKxUTf)

<img src="https://github.com/ant-media/Ant-Media-Server/assets/3456251/733523cf-edc3-4eda-a8ba-6a93f4a16fb8" width=500px/>

Live Coding: How to Develop WebRTC iOS apps?
Friday, May 19 · 2:00 – 2:30pm
Time zone: Europe/Istanbul
Video call link: https://meet.google.com/wuh-raap-nbu

[Add event to your calendar](https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=NTRqdDgzNTBlc29pNWcyN2dsYjM3cjFodHYgbWVreWFAYW50bWVkaWEuaW8&tmsrc=mekya%40antmedia.io) 

<a target="_blank" href="https://calendar.google.com/calendar/event?action=TEMPLATE&amp;tmeid=NTRqdDgzNTBlc29pNWcyN2dsYjM3cjFodHYgbWVreWFAYW50bWVkaWEuaW8&amp;tmsrc=mekya%40antmedia.io"><img border="0" src="https://www.google.com/calendar/images/ext/gc_button1_en.gif"></a>

**Comment by fullakingi** [comment]
This looks awesome.

**Comment by fullakingi** [comment]
wow that is a super helpful app - thank you. I will get it going ;)

**Comment by asharamseervi** [comment]
Thank you to everyone who joined our webinar on "**How to Develop WebRTC iOS apps**"!

We hope you enjoyed the live coding session and learned some awesome stuff.

> In case you missed it or want to watch it again; you can find the recording here: https://www.youtube.com/watch?v=gy9jZH-qyxA

We appreciate your feedback and questions and look forward to seeing you in our next webinar. Stay tuned for more updates and announcements on our community discussion forum. Happy coding! 😊
